src/routes/landing/components/CityEntry.js

```diff
@@ -31,11 +34,31 @@ type PropsType = {|
 class CityEntry extends React.PureComponent<PropsType> {
   render () {
     const { city, language, filterText } = this.props
+    const getMunicipality = (city, filterText): Array<CityModel> => {
+      if (!filterText) {
+        return []
+      }
+      if (city._aliases) {
+        const municipalities = Object.keys(city._aliases).map(municipality => municipality.toLowerCase())
+        const municipalityMatch = municipalities.filter(municipality => municipality.includes(filterText.toLowerCase()))
+        const municipalityMatchCapitalized = municipalityMatch.map(match => match[0].toUpperCase() + match.slice(1))
+        return municipalityMatchCapitalized
```

Comment:
I'd prefer chaining the calls (with comments if unclear whats happening). Also, I think it is not guaranteed that municipalities are always containing only one word, so your approach on recapitalizing is probably not enough. I'd rather don't lowercase it in the first place, so you don't have to revert it later on.
```suggestion
        return Object.keys(city._aliases)
             .filter(municipality => municipality.toLowerCase().includes(filterText.toLowerCase()))
        return municipalityMatchCapitalized
```